app/views/pages/home.html.haml

```diff
@@ -67,12 +67,36 @@
         .small-12.medium-6.columns
           = link_to 'Start a Hack Club', apply_path, class: 'button outline-outward'
 #home-map-jumbotron
+.main-section.sponsors
+  %h1 Sponsors
+  .row.logos
+    .medium-12.large-6.columns
+      = link_to "https://codehs.com/" do
+        %img{alt: 'CodeHS', src: '/assets/codehs.png'}
+      = link_to "http://www.hackreactor.com/" do
+        %img{alt: 'Hack Reactor', src: '/assets/hack_reactor.png'}
+      = link_to "https://workflowy.com/" do
+        %img{alt: 'Workflowy', src: '/assets/workflowy.png'}
+      = link_to "http://firstcodeacademy.com/" do
+        %img{alt: 'First Code Academy', src: '/assets/first_code_academy.png'}
+    .medium-12.large-6.columns
+      = link_to "https://www.makeschool.com" do
+        %img{alt: 'Make School', src: '/assets/make_school.svg'}
+      = link_to "https://www.nitrous.io/" do
+        %img{alt: 'nitrous.io', src: '/assets/nitrousio.svg'}
```

Comment:
Will this work with the Rails asset pipeline? You may need to use the `image_tag`.